unstructured/ingest/main.py

```diff
@@ -103,6 +133,20 @@ def run(self):
 
 
 @click.command()
+@click.option(
+    "--metadata-include",
+    cls=MutuallyExclusiveOption,
+    default="",
+    help="If set, include the specified metadata fields if they exist and drop all other fields.",
```

Comment:
How does one assign this list when calling? Is it a single string with comma separated values? or do you make multiple `--metadata-include X --metadata-include Y` args? Maybe helpful to have an example in the help. (Same for exclude)